modules/tensor_mechanics/include/userobjects/TensorMechanicsPlasticTensileMulti.h

```diff
@@ -173,18 +179,18 @@ class TensorMechanicsPlasticTensileMulti : public TensorMechanicsPlasticModel
  protected:
   const TensorMechanicsHardeningModel & _strength;
 
-  /// whether to use the custom return-map algorithm
-  bool _use_custom_returnMap;
-
-  /// whether to use the custom consistent tangent operator algorithm
-  bool _use_custom_cto;
-
   /// maximum iterations allowed in the custom return-map algorithm
   unsigned int _max_iters;
 
   /// yield function is shifted by this amount to avoid problems with stress-derivatives at equal eigenvalues
   Real _shift;
```

Comment:
This should also be const. It helps people reading the code to make clear that this member is not changed after construction, and it helps avoid bugs in the code by accidentally changing these members.